store/src/migrations.rs

```diff
@@ -3,7 +3,29 @@ use rusqlite_migration::{Migrations, M};
 
 pub static MIGRATIONS: Lazy<Migrations> = Lazy::new(|| {
     Migrations::new(vec![M::up(
-        "CREATE TABLE nullifiers (id INTEGER PRIMARY KEY, nullifier BLOB, block_number INTEGER);",
+        "
+        CREATE TABLE
+            block_header
+        (
+            block_num INTEGER NOT NULL,
+            serialized_block_header BLOB NOT NULL,
```

Comment:
nit: should we call this just `block_header`? The fact that it is serialized is kind of implied.

Reply:
renamed